packages/view/backend/sirius-components-view-emf/src/main/java/org/eclipse/sirius/components/view/emf/diagram/ViewDiagramDescriptionConverter.java

```diff
@@ -109,10 +112,10 @@ public class ViewDiagramDescriptionConverter implements IRepresentationDescripti
         return UUID.nameUUIDFromBytes(EcoreUtil.getURI(diagramElementDescription).toString().getBytes());
     };
 
-    public ViewDiagramDescriptionConverter(IObjectService objectService, IEditService editService) {
+    public ViewDiagramDescriptionConverter(IObjectService objectService, IEditService editService, StylesFactory stylesFactory) {
```

Comment:
Ask `List<INodeStyleProvider>` here and give it to `StylesFactory` instead of making the StylesFactory a service.

Reply:
I will change that